packages/utils/package-manager.ts

```diff
@@ -13,17 +13,18 @@ import * as path from "path";
  */
 
 export function getPackageManager(): string {
-	const hasLocalNPM = fs.existsSync(path.resolve(process.cwd(), "package-lock.json"));
-	const hasLocalYarn = fs.existsSync(path.resolve(process.cwd(), "yarn.lock"));
-	if (hasLocalNPM) {
-		return "npm";
-	} else if (hasLocalYarn) {
-		return "yarn";
-	} else if (spawn.sync("yarn", [" --version"], { stdio: "ignore" }).error) {
-		return "npm";
-	} else {
-		return "npm";
-	}
+    const pwd = path.resolve(process.cwd());
+    const hasLocalNPM = fs.existsSync(path.resolve(pwd + "/package-lock.json"));
+    const hasLocalYarn = fs.existsSync(path.resolve(pwd + "/yarn.lock"));
+    if (hasLocalNPM) {
+        return "npm";
+    } else if (hasLocalYarn) {
+        return "yarn";
+    } else if (spawn.sync("yarn", [" --version"], { stdio: "ignore" }).error) {
```

Comment:
Here we are returning "npm" in else block too, maybe this if can be removed?

Reply:
I thought of that too, here else works like a default case. What about removing the check for package-lock.json, as default is npm, so why check for it.

Reply:
Yes, I think what you're suggesting would be better, just checking for `yarn` if yarn is not there then return `npm`.

Reply:
Have a look at the recent changes. Any idea about the failing tests.